9-alphabetical_order.py

```diff
@@ -0,0 +1,7 @@
+def alphabetical_order():
+    items = [n for n in input().split('-')]
+    #print(items)
+    items.sort()
+    print('-'.join(items))
+
+alphabetical_order()
```

Comment:
- kodlarınız çalışıyor tebrikler. input un içine mesaj yazmayı unutmuşsunuz.
for dongüsü ile input almak güzel fikir. ayrıca items = list(input().split('-')) , bu şekildede alabilirsiniz. bu şekilde hiç döngüye girmemiş olur.
- açıklama eklerseniz güzel olur.